about.html

```diff
@@ -4,13 +4,17 @@
 slug: about
 ---
 <p>
-	LXQt is a lightweight Desktop Environment. It is the product of the merge between the LXDE-Qt and the Razor-qt projects: A lightweight, modular, blazing-fast and user-friendly desktop environment.
+LXQt is a lightweight Qt desktop environment.
 </p>
 <p>
-	LXQt will not replace <a href="http://lxde.org">LXDE</a>. They are both active and maintained projects.  See the <a href="https://github.com/lxde/lxqt/wiki/History">history</a> section in our wiki to learn more about the relation between these two desktop environments.
+It will not get in your way. It will not hang or slow down your system. It is focused on being a classic desktop with a modern look and feel.
 </p>
 <p>
-	<strong>LXQt 0.10.0</strong> is a fast and stable desktop environment already usable on production systems.<br/>
-	It will not get in your way. It will not hang or slow down your system. It is focused on being a <em>Classic Desktop</em> with a modern Look &amp; Feel.<br/>
-	Already available on most Linux and BSD distributions. Try it out at home or in a VM!
+LXQt has already been included in most Linux and BSD distributions so you may just try it out on your regular system or in a VM.<br>
+More information about installing can be found in the <a href="https://github.com/lxde/lxqt/wiki/">LXQt GitHub wiki</a>.
+</p>
+<p>
+Historically, LXQt is the product of the merge between LXDE-Qt, an initial Qt flavour of LXDE, and Razor-qt, a project aiming to develop a Qt based desktop environment with similar objectives as the current LXQt.<br>
+LXQt was first supposed to become the successor of LXDE one day but as of 09/2016 both desktop environments will keep coexisting for the time being.<br>
+More on this topic can be found in page <a href="https://github.com/lxde/lxqt/wiki/History">History</a> of the aforementioned wiki.
```

Comment:
Non native speaker here: but it's possible that it should be `can be found on page` on instead of in. Maybe it should also be `can be found on the page History`